template.html

```diff
@@ -1,2 +1,2 @@
-angular.module("<%= moduleName %>"<% if (createModule) { %>, []<% } %>)<% _.forEach(constants, function (constant) { %>
-.<%= type %>("<%= constant.name %>", <%= constant.value %>)<% }); %>;
+angular.module('<%= moduleName %>'<% if (createModule) { %>, []<% } %>)<% _.forEach(constants, function (constant) { %>
+.<%= type %>('<%= constant.name %>', <%= constant.value %>)<% }); %>;
```

Comment:
We should probably keep things backward compatible - maybe this could be a setting which defaults to double quotes

```js
gulpNgConfig('config', {
  singleQuotes: true
})
```